plenoptic/simulate/models/frontend.py

```diff
@@ -102,14 +420,52 @@ def display_filters(self, zoom=5., **kwargs):
         fig: PyrFigure
         """
 
-        weights = torch.cat([self.linear.weight.detach(),
-                             self.luminance.weight.detach(),
-                             self.contrast.weight.detach()], dim=0)
+        weights = torch.cat(
+            [
+                self.on.center_surround.filt,
+                self.off.center_surround.filt,
+                self.on.luminance.filt,
+                self.off.luminance.filt,
+                self.on.contrast.filt,
+                self.off.contrast.filt,
+            ],
+            dim=0,
+        ).detach()
 
-        title = ['linear on', 'linear off',
-                'luminance norm on', 'luminance norm off',
-                'contrast norm on', 'contrast norm off']
+        title = [
+            "linear on",
+            "linear off",
+            "luminance norm on",
+            "luminance norm off",
+            "contrast norm on",
+            "contrast norm off",
```

Comment:
same comment about luminance / contrast norm as above

Reply:
Changed to saying filt instead of norm